components/automate-cli/cmd/chef-automate/applications.go

```diff
@@ -46,6 +73,65 @@ func newApplicationsDisableCmd() *cobra.Command {
 	}
 }
 
+func runApplicationsListDisconnectedServicesCmd(*cobra.Command, []string) error {
+	if thresholdMinutes <= 0 {
+		return status.Errorf(status.InvalidCommandArgsError,
+			"%d is not a valid threshold time in minutes. The expected time must be greater than zero.",
+			thresholdMinutes,
+		)
+	}
+
+	if format != "json" && format != "pretty" {
+		return status.Errorf(status.InvalidCommandArgsError,
+			"%s is not a valid format type. Available formats are 'json' and 'pretty'.",
+			format,
+		)
+	}
+
+	var (
+		ctx            = context.Background()
+		apiClient, err = apiclient.OpenConnection(ctx)
+	)
+	if err != nil {
+		return status.Wrap(err, status.APIUnreachableError,
+			"Failed to create a connection to the API")
+	}
+
+	servicesRes, err := apiClient.ApplicationsClient().GetDisconnectedServices(ctx,
+		&applications.DisconnectedServicesReq{
+			ThresholdMinutes: int32(thresholdMinutes),
+		},
+	)
+	if err != nil {
+		return err
```

Comment:
Should we wrap this error like the others? Similar question about some of the error returns below.